nixos/modules/hardware/video/webcam/ipu6.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+
+  cfg = config.hardware.ipu6;
+
+in
+{
+
+  options.hardware.ipu6 = {
+
+    enable = mkEnableOption (lib.mdDoc "ipu6 kernel module");
+
+    platform = mkOption {
+      default = "ipu6";
```

Comment:
I think we shouldn't set a default here.

Reply:
done